build.gradle

```diff
@@ -1,12 +1,13 @@
+import org.ajoberstar.grgit.Grgit
 import org.gradle.util.VersionNumber
 
 buildscript {
-    ext.android_tools = 'com.android.tools.build:gradle:4.1.0'
+    ext.android_tools = 'com.android.tools.build:gradle:7.4.0'
     ext.errorproneVersion = '2.4.0'
     ext.errorproneJavacVersion = '9+181-r4173-1'
```

Comment:
Down the road we should get rid of extension properties

Reply:
SGTM